src/clj/harja/kyselyt/uusi_kustannussuunnitelma_kyselyt.clj

```diff
@@ -547,7 +549,12 @@
                          :vuosi hoitovuoden-alkuvuosi
                          :kuukausi kk
                          :summa summa
-                         :summa_indeksikorjattu nil
+                         :summa_indeksikorjattu (when summa
+                                                  (indeksi-kyselyt/indeksikorjaa
+                                                    (indeksi-kyselyt/indeksikerroin urakan-indeksit
+                                                      (pvm/paivamaara->mhu-hoitovuosi-nro
+                                                        (:alkupvm urakan-tiedot) (pvm/luo-pvm-dec-kk hoitovuoden-alkuvuosi kk 1)))
```

Comment:
En heti tajunnu miksi kuukaudesta vähennetään tässä yksi. Indeksilukutaulukosta haetaan edellisen kuukauden luku, jota käytetään indeksikertoimen laskennassa?  

Nää kannattaa koodata jotenkin silleen, että on helppo löytää muutettava paikka, jos indeksikertoimen laskentasääntö muuttuu. Tätä ei kannattais koodata niin, että on vuodelle 2025 omia ratkaisuja, kaikkien indeksikerroinlaskentojen pitäis löytyä samasta paikasta, koska niiden kanssa menee helposti sekaisin muutenkin. 

Sääntö indeksiluvusta on 2023 alkaen ollut, että käytetään elokuun indeksilukua (ks. tietokantaproseduuri laske_kuukauden_indeksikorotus_urakalle). Ilmeisesti 2025 tulee taas uusi sääntö. Nää säännöt pitäis mun mielestä näkyä samassa paikassa.